app/views/shared/views/ems_common/angular/_form.html.haml

```diff
@@ -138,6 +153,18 @@
                      "checkchange"                 => "",
                      "selectpicker-for-select-tag" => "")
 
+    .form-group{"ng-class" => "{'has-error': angularForm.azure_stack_api_version.$invalid}", "ng-if" => "emsCommonModel.emstype == 'azure_stack'"}
```

Comment:
You have to use `angularForm.api_version.$invalid` instead of `angularForm.azure_stack_api_version.$invalid`.

Reply:
Done